Solutions/IoTOTThreatMonitoringwithDefenderforIoT/Analytic Rules/IoTDenialofService.yaml

```diff
@@ -16,9 +16,9 @@ tactics:
   - Impact
```

Comment:
There is no technique mapped for this Analytical rule.

Reply:
hi @devikamehra, not sure what's the problem. Could you please clarify?